platform/ios/resources/zh-Hans.lproj/Localizable.strings

```diff
@@ -0,0 +1,84 @@
+/* Accessibility hint */
+"ANNOTATION_A11Y_HINT" = "显示信息";
+
+/* No comment provided by engineer. */
+"API_CLIENT_400_DESC" = "The session data task failed. Original request was: %@";
```

Comment:
These error messages come from Mapbox Telemetry. In general, error messages should be translated whenever possible. However, in this case, they aren’t ever presented to the user, so it’s fine to leave these messages in English.